### PR TITLE
ReadMe Proofreading

### PR DESCRIPTION
**Pull request by rayliverified**
Collection of ReadMe proofreading changes to remediate Issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ As you can see `get()` is used exactly like using `GetIt` directly with all its
 **get() and getX() can be called multiple times inside a Widget and also outside the `build()` function.**
 
 ### Watching Data
-The following functions will return a value and rebuild the widget every-time this data inside GetIt changes. **Important: This function can only be called inside the `build()` function and you can only watch any objects only once. Also all of these function have to be called always and in the same order on every `build` meaning they can't be called conditionally otherwise the mixin gets confused**
```
Comment (escamoteur) [inline README.md:70]: the 'in the same order' is important. could you add that please?

**Review by escamoteur — CHANGES_REQUESTED** [review]
Thanks a lot. if you could make the small change I merge it. Thanks again

**Review by escamoteur — APPROVED** [review]
awesome. you are a great help!

**Comment by rayliverified** [comment]
Thank you! Will create test cases to demo the issues. Didn't have enough time last night so apologies for the incomplete issues.

**Comment by escamoteur** [comment]
what is your twitter handle again?

**Comment by rayliverified** [comment]
@rayliverified

**Pull request merged**